subsys/canbus/isotp/isotp.c

```diff
@@ -76,7 +76,7 @@ static inline void receive_report_error(struct isotp_recv_ctx *ctx, int err)
 	ctx->error_nr = err;
 }
 
-void receive_can_tx_isr(int error, void *arg)
+void receive_can_tx(int error, void *arg)
```

Comment:
```suggestion
static void receive_can_tx(int error, void *arg)
```
That's my fault. Sorry

Reply:
Fixed.